src/mon/MonitorDBStore.h

```diff
@@ -736,6 +743,61 @@ class MonitorDBStore
     db.reset(NULL);
   }
 
+  /// @brief Creates a backup of the database
+  /// @param backup_path location to create backup at
+  /// @return true on success
+  KeyValueDB::BackupStats backup(bool full = false) {
```

Comment:
Does this actually need a default value?

Reply:
When full is not set, incremental backup is likely created.